test/signup.test.js

```diff
@@ -0,0 +1,168 @@
+const request = require('supertest'),
+  app = require('../app.js'),
+  Users = require('../app/models').user,
+  { factory } = require('factory-girl'),
+  validationErrorStatus = 401,
+  createdCorrectlyStatus = 200,
+  firstName = 'fn',
+  lastName = 'ln',
+  correctPassword = 'password',
+  correctEmail = 'email@wolox.com.ar';
+
+describe('Test invalid password', () => {
```

Comment:
I like how you grouped the tests in the describes, but remember you should also add a bigger group by the endpoint itself (`describe('POST /users', ...`).